### ISSUE TITLE
Implement Mouse Wheel Camera

### ISSUE THREAD

**Issue by HelloKitty** [HelloKitty/RS317.Sharp#4]
An implementation can be seen here: https://www.rune-server.ee/runescape-development/rs2-client/tutorials/552017-middle-mouse-movement.html

**Comment by ghost** [comment]
yes please! for the love, why wasn't this in the official client. :-) i'd love WASD moving too, but that gets tricky.

[edit] just realized, maybe scroll wheel wasn't standard back then?

**Comment by HelloKitty** [comment]
Yea I agree, I might give it a shot today. It's definitely a missed addition but I've so much things I have to work on atm it might sit on the backburner.
